src/test/java/be/syntra/devshop/DevshopBack/factories/UserFactoryTest.java

```diff
@@ -0,0 +1,41 @@
+package be.syntra.devshop.DevshopBack.factories;
+
+import be.syntra.devshop.DevshopBack.entities.User;
+import be.syntra.devshop.DevshopBack.security.entities.UserRole;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.List;
+import java.util.Optional;
+
+import static be.syntra.devshop.DevshopBack.security.entities.UserRoles.ROLE_USER;
+import static be.syntra.devshop.DevshopBack.testutilities.UserUtils.createUser;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+public class UserFactoryTest {
+
+    @Mock
+    private UserFactory userFactory;
+
+    @BeforeEach
+    public void init() {
+        MockitoAnnotations.initMocks(this);
+    }
```

Comment:
pls don't mock the class you're testing ;-)

Reply:
Fixed